src/dynarec/rv64/dynarec_rv64_00_0.c

```diff
@@ -317,7 +321,27 @@ uintptr_t dynarec64_00_0(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
             MOV64xw(x2, i64);
             emit_sbb32(dyn, ninst, rex, xRAX, x2, x3, x4, x5);
             break;
-
+        case 0x1E:
+            if (rex.is32bits) {
+                INST_NAME("PUSH DS");
+                LHU(x1, xEmu, offsetof(x64emu_t, segs[_DS]));
+                PUSH1_32(x1);
+                SMWRITE();
+            } else {
+                DEFAULT;
+            }
+	    break;
+        case 0x1F:
+            if (rex.is32bits) {
+                INST_NAME("POP DS");
+                SMREAD();
+                POP1_32(x1);
+                SH(x1, xEmu, offsetof(x64emu_t, segs[_DS]));
+                SW(x1, xEmu, offsetof(x64emu_t, segs_serial[_DS]));
```

Comment:
Save x1 to `segs_serial` does not make sense to me, other `POP`s implemented in previous PR share the same issue.

Reply:
Looking at ARM64 DynaRec, it's saving xZR to `segs_serial` to indicate there need a re-compute on the segment offsets. We should do the same.